ament_ikos/ament_ikos/main.py

```diff
@@ -161,9 +161,31 @@ def aggregate_junit_xml_files(ikos_db_filenames, summary_filename, summary_name)
 
 
 def aggregate_sarif_files(ikos_db_filenames, summary_filename, summary_name):
-    """Aggregate the SARIF files for each test into a single ikos.sarif file."""
-    # TODO(mjeronimo): Write a function to aggregate the SARIF files
-    pass
+    """Aggregate the SARIF files for each test into a single output SARIF file."""
+    first = True
+    output_dict = {}
+
+    for db_filename in ikos_db_filenames:
+        # Generate the input SARIF filename from the IKOS db name
+        sarif_filename = os.path.splitext(db_filename)[0] + '.sarif'
+
+        # Process the file
+        with open(sarif_filename) as input_file:
+            data = json.load(input_file)
+            # Grab the version and schema information from the first file
+            if first:
+                output_dict['version'] = data['version']
+                output_dict['$schema'] = data['$schema']
+                output_dict['runs'] = []
+                first = False
```

Comment:
What if the info doesn't match subsequent files? Should the `else` case exist to check that?

Reply:
I thought about it, but since this the output files are from an ament_ikos run, ament_ikos will be invoking ikos multiple times, once for each executable. In other words, the files will just have been generated using the same ikos. I can at least add a warning, I suppose, if I find a difference.

Reply:
I think it's worth just asserting, so you could catch some weird case you didn't expect, like two ikos runs ending up in the same directory or something.

Reply:
Done.